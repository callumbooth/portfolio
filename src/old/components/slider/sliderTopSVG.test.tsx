import React from "react";
import { render } from "@testing-library/react";
import SliderTopSVG from "./sliderTopSVG";

it("Svg rotates to 45 deg correctly", () => {
  const { container } = render(<SliderTopSVG rotation={45} />);

  expect(container).toMatchSnapshot();
});

it("Svg rotates by 90 deg correctly", () => {
  const { container } = render(<SliderTopSVG rotation={90} />);

  expect(container).toMatchSnapshot();
});
