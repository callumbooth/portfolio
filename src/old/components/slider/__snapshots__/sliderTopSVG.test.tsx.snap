// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Svg rotates by 90 deg correctly 1`] = `
<div>
  <svg
    viewBox="0 0 1520 1080"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <line
        class="stroke-current text-primary-main stroke-3"
        fill="none"
        pathLength="100"
        stroke-dasharray="100"
        stroke-dashoffset="100"
        transform="translate(1090 0) rotate(90 217.5 217.5)"
        x2="435"
        y2="435"
      />
      <line
        class="stroke-current text-primary-main stroke-3"
        fill="none"
        pathLength="100"
        stroke-dasharray="100"
        stroke-dashoffset="100"
        transform="translate(417 0) rotate(90 184.5 184.5)"
        x2="369"
        y2="369"
      />
    </g>
  </svg>
</div>
`;

exports[`Svg rotates to 45 deg correctly 1`] = `
<div>
  <svg
    viewBox="0 0 1520 1080"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <line
        class="stroke-current text-primary-main stroke-3"
        fill="none"
        pathLength="100"
        stroke-dasharray="100"
        stroke-dashoffset="100"
        transform="translate(1090 0) rotate(45 217.5 217.5)"
        x2="435"
        y2="435"
      />
      <line
        class="stroke-current text-primary-main stroke-3"
        fill="none"
        pathLength="100"
        stroke-dasharray="100"
        stroke-dashoffset="100"
        transform="translate(417 0) rotate(45 184.5 184.5)"
        x2="369"
        y2="369"
      />
    </g>
  </svg>
</div>
`;
